contracts/Chaosnet.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity 0.8.9;
+
+/// @title Chaosnet
+/// @notice This is a beta staker program for stakers willing to go the extra
+/// mile with monitoring, share their logs with the dev team, and allow to more
+/// carefully monitor the bootstrapping network. As the network matures, the
+/// beta program will be ended.
+contract Chaosnet {
+  /// @notice Indicates if the chaosnet is active. The chaosnet is active
+  /// after the contract deployment and can be ended with a call to
+  /// `deactivateChaosnet()`. Once deactivated chaosnet can not be activated
+  /// again.
+  bool public isChaosnetActive;
+
+  /// @notice Indicates if the given operator is a beta operator for chaosnet.
+  mapping(address => bool) public isBetaOperator;
+
+  /// @notice Address controlling chaosnet status and beta operator addresses.
+  address public chaosnetOwner;
+
+  event BetaOperatorsAdded(address[] operators);
+
+  event ChaosnetOwnerRoleTransferred(
+    address oldChaosnetOwner,
+    address newChaosnetOwner
+  );
+
+  event ChaosnetDeactivated();
+
+  constructor() {
+    _transferChaosnetOwner(msg.sender);
+    isChaosnetActive = true;
+  }
+
+  modifier onlyChaosnetOwner() {
+    require(msg.sender == chaosnetOwner, "Not the chaosnet owner");
+    _;
+  }
+
+  modifier onlyOnChaosnet() {
+    require(isChaosnetActive, "Chaosnet is not active");
+    _;
+  }
+
+  /// @notice Adds beta operator to chaosnet. Can be called only by the
+  /// chaosnet owner when the chaosnet is active.
```

Comment:
Can we add a comment that once an operator is added it cannot be removed?

Reply:
82d14dfbc04155f3dc01dcfdbd5f638fda27fc39